### PR TITLE
Bugfix/enable Windows CI unit tests and test coverage

### PR DESCRIPTION
**Pull request by allexzander**

### CODE REVIEW

File: .github/workflows/sonarcloud.yml [hunk 1]
```diff
@@ -48,3 +48,82 @@ jobs:
...
+          }
+          
+          runTestsAndCreateCoverage
+
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          directory: ${{ github.workspace }}\cobertura_coverage
+          fail_ci_if_error: true
```
[the remark this answers is not in the document]
Comment (allexzander) [reply .github/workflows/sonarcloud.yml:129]: @mgallien Will do

File: .github/workflows/sonarcloud.yml [hunk 2]
```diff
@@ -48,3 +48,82 @@ jobs:
...
+          
+          function craft() {
+              python "${{ env.CRAFT_MASTER_LOCATION }}\CraftMaster.py" --config "${{ env.CRAFT_MASTER_CONFIG }}" --target ${{ env.CRAFT_TARGET }} -c $args
+              if($LASTEXITCODE -ne 0) {exit $LASTEXITCODE}
+          }
+          
+          craft --add-blueprint-repository [git]https://github.com/nextcloud/desktop-client-blueprints.git
+          craft craft
+          craft --install-deps nextcloud-client
+          craft nsis
```
Comment (mgallien) [inline .github/workflows/sonarcloud.yml:78]: remove this we no longer build a package with nsis

**Comment by allexzander** [comment]
@mgallien All done

**Review by mgallien — APPROVED** [review]
nice !

**Review by claucambra — APPROVED** [review]

**Pull request merged**